src/main/resources/templates/fragments/header.html

```diff
@@ -3,12 +3,15 @@
 <head>
 </head>
   <body>
-    <div th:fragment="header">
-    	<nav class="navbar navbar-expand-sm bg-light">
+    <div class="navbar navbar-expand-sm bg-light" th:fragment="header">
```

Comment:
Moved the nav tag into the div tag, no need for a nav tag when the class id can be included in the div tag